tests/integration/src/test/scala/mesosphere/marathon/integration/UpgradeIntegrationTest.scala

```diff
@@ -273,16 +277,20 @@ class UpgradeIntegrationTest extends AkkaIntegrationTest with MesosClusterTest w
     marathonCurrent.close()
   }
 
-  def killTask(appName: String): Unit = {
-    val pidPattern = """([^\s]+)\s+([^\s]+)\s+.*""".r
-    val pids = Process("ps aux").!!.split("\n").filter { process =>
-      process.contains("src/test/resources/python/app_mock.py") && process.contains(appName)
-    }.collect {
-      case pidPattern(_, pid) => pid
+  def suicideTasks(tasks: List[ITEnrichedTask]): Unit = {
+    logger.info(s"Sending suicide requests to the tasks of the ${tasks.head.appId}: ${tasks.map(_.id)}")
+    tasks.foreach{ task =>
+      val host = task.host
+      val ports = task.ports.headOption.value
+      val port = ports.headOption.value
+
+      val url = Uri.from(scheme = "http", host = host, port = port, path = "/suicide")
+      Http().singleRequest(Get(url)).map { result =>
```

Comment:
What do you think about making this a `Delete(url)`?

Reply:
You so smart, @jeschkies ! Done!